### PR TITLE
Bug 1245058 - Add vendor blob list r=gerard-majax

### PR DESCRIPTION
**Pull request by atilag**

### CODE REVIEW

File: FP2-blobs-out.txt [hunk 1]
```diff
@@ -0,0 +1,427 @@
+target/product/FP2/system/vendor/lib/libmmipstreamsourcehttp.so
```
Comment (lissyx) [inline FP2-blobs-out.txt:1]: That file is not enough. You need to add:
- TARGET_DEVICE_BLOBS :=  FP2-blobs-out.txt
- TARGET_DEVICE_BLOBS_LIST := true

Comment (stellirin) [inline FP2-blobs-out.txt:1]: I dont see your vendor include anywhere in the repo so the 2nd line is just my assumption:

```
TARGET_DEVICE_BLOBS_LIST := true
TARGET_DEVICE_BLOBS_SRC_DIR := vendor/fairphone/FP2/proprietary
TARGET_DEVICE_BLOBS := device/fairphone_devices/FP2/proprietary-files.txt
```

The proprietary files can just be a list of the files in their expected locations:

proprietary-files.txt:
`vendor/lib/libmmipstreamsourcehttp.so`

Comment (stellirin) [inline FP2-blobs-out.txt:1]: Just to be sure, these are used for making blobfree buids, is that what you are trying to do here?

Comment (atilag) [inline FP2-blobs-out.txt:1]: Yes it is. Anyway I'm going to close this PR and make a new one which I tested from scratch and worked fine.

**Pull request closed without merging**